test/toHaveUniqueItems.spec.js

```diff
@@ -4,13 +4,13 @@ describe("toHaveUniqueItems", function() {
     it("should accept Array in actual", function() {
         expect(function() {
             expect("test").not.toHaveUniqueItems();
-        }).toThrowError("Actual is not Array")
+        }).toThrowError("Actual is not Array. It is of type: " + typeof("test"))
```

Comment:
I missed that this comment was in the test. I guess that makes more sense. I'll change in and push up again.